en/docs/troubleshoot/faq.md

```diff
@@ -0,0 +1,147 @@
+# Frequently Asked Questions
+
+## General
+
+### Q: What is Choreo?
+Choreo is your SaaS application development suite designed to accelerate the creation of digital experiences. With Choreo you can build, deploy, monitor, and manage your cloud-native applications while increasing developer productivity, allowing you to focus on innovation.
+
+Choreo consists of five profiles:
```

Comment:
We did away with profiles so we shouldn't have this info I suppose.

Reply:
We still do have the marketing profiles. Removed that sections that mentions about profiles from the console.